Makefile

```diff
@@ -5,7 +5,7 @@ TAG=release-$(shell date +%s)
 local.update:
 	@if ! command -v mass  &> /dev/null; then echo "please install massdriver cli https://github.com/massdriver-cloud/massdriver-cli" && exit 1; fi
 	@mkdir -p .artifacts
-	@rm .artifacts/*
+	@rm -f .artifacts/*
```

Comment:
this is intentional change as this make target failed on the first use for me because `.artifacts/` was empty the -f will workaround that edge case.